### PR TITLE
fix(sync): try to reconnect to stream server when the conn is disconn…

### PR DESCRIPTION
**Pull request by zibuyu28**
if the stream goroutine is stop because the network or other error, the sync work will be block because there are no logic will renew a stream worker.

### CODE REVIEW

**Comment by zibuyu28** [comment]
installed

File: zk/datastream/client/stream_client.go [hunk 1]
```diff
@@ -286,6 +291,18 @@ func (c *StreamClient) readAllFullL2BlocksToChannel() error {
 	return err
 }
 
+func (c *StreamClient) tryReConnect() {
+	for {
```
Comment (mandrigin) [inline zk/datastream/client/stream_client.go]: let's make 5 retries

Comment (zibuyu28) [reply zk/datastream/client/stream_client.go]: done for this

**Comment by revitteth** [comment]
@cla-bot check

**Pull request merged**